.github/actions/init/action.yml

```diff
@@ -1,11 +1,11 @@
 name: Install deps
-description: ""
+description: ''
 runs:
   using: composite
   steps:
-    - uses: actions/setup-node@v2
+    - uses: actions/setup-node@v4
```

Comment:
`v4` is the latest major version of `actions/setup-node` at the time of this writing.